solutions.js

```diff
@@ -1,5 +1,41 @@
 console.log('Defining functions!');
 
+//Create a function that changes the navbar to a different color.
 function changeNavColor() {
-  // change the navbar to a different color
+  document.getElementById("topbar").style["background-color"] = "red";
 };
+
+//Add a new <h1> to the top of the page that says <h1>This page is now
+//under *YOUR NAME*'s control!'</h1>.
+function addHeader() {
+  //document.querySelector("body").innerHtml = "<h1>This page is now under Jay's control!'</h1>" +document.querySelector("body").innerHtml;
+  document.querySelector("body").insertAdjacentHTML("afterbegin","<h1>This page is now under Jay's control!'</h1>");
```

Comment:
Looks like there is an extra `'` at the end of your text. You'll want to make sure you are escaping those quotes if they aren't different than the string quotes. I'm guessing this was just a typo though.